lib/media/track/remotemediatrack.js

```diff
@@ -8,7 +9,8 @@ function mixinRemoteMediaTrack(AudioOrVideoTrack) {
   /**
    * A {@link RemoteMediaTrack} represents a {@link MediaTrack} published to a
    * {@link Room} by a {@link RemoteParticipant}.
-   * @property {boolean} isEnabled - Whether the {@link RemoteMediaTrack} is enabled
+   * @property {boolean} isEnabled - <code>Deprecated: Use (.isSwitchedOff && .switchOffReason === "disabled-by-publisher") instead</code>
```

Comment:
I'm guessing `isEnabled` implies either switched on, or switched off for other reason than 'disabled-by-publisher'? Should this be:
```suggestion
   * @property {boolean} isEnabled - <code>Deprecated: Use (.switchOffReason !== "disabled-by-publisher") instead</code>
```
(ditto in other places.)